README: htmlify, update links (#1745)
authzed/spicedb: README.md

If you don't want to start with the examples loadable from the Playground, you can follow a guide for [developing a schema] or review the the schema language [design documentation].

> To aquire an understanding of schema development, you can watch this primer video:

Reviewer: Typo "acquire"

```suggestion
Watch the SpiceDB Primer video to get started with schema development.
```

In addition, we'd like to highlight a few notable contributions:

- The GitHub Authorization Team for implementing and contributing the MySQL datastore

> - The Netflix Authorization Team for sponsoring and being a design partner for caveats

Reviewer: Can we add logos or at least favicon size images next to GitHub and Netflix?

Author: IDK if it looks like an improvement:

<img width="803" alt="Screenshot 2024-02-18 at 8 53 37 PM" src="https://github.com/authzed/spicedb/assets/343539/80f10961-a402-4324-9082-7ce8ae351b8c">
